fedora/ovn/Dockerfile

```diff
@@ -13,7 +13,13 @@ COPY install_ovn.sh /install_ovn.sh
 
 RUN /install_ovn.sh $USE_OVN_RPMS
 
-RUN dnf -y remove automake make gcc autoconf openssl-devel libtool
+# Generate variation of dhclient-script that we can use for fake vm namespaces
+RUN dnf -y install dhclient which
```

Comment:
Thanks for the PR. This is really cool.

One comment. Do you think its good to move the dhclient stuff to fedora/Dockerfile ?

The main advantage I see is that, the base image will have dhclient installed and later on If I want to just update the OVN rpms or sources, the build will be relatively faster.

I am fine if this stuff makes sense here :).
Thoughts ?

Reply:
I mean fedora/cinc/Dockerfile.